### DISCUSSION TITLE
Bounding the output function from PySR

### DISCUSSION THREAD

**Discussion by Khansolo9** [MilesCranmer/PySR#211] [Q&A] [answered]
Although I am successfully getting a function for our data - as expected - is there any way I could get an inequality for our data as an output of PySR, as well? Meaning, is there anyway I could bound the data …? E.g. if expected value calculated is <X, then satisfied, otherwise, not.

Best,
Khansolo9

P.S. thoroughly enjoy using this software!

**Comment by MilesCranmer** [comment, marked as answer]
Thanks!

Like this?
```python
model = PySRRegressor(loss="loss(x, y) = (x <= y) ? ((x-y)^2) : Inf")
```

**Reply by MilesCranmer** [reply]
Although I might choose a large number rather than infinity. i.e., 
```python
model = PySRRegressor(loss="loss(x, y) = (x <= y) ? ((x-y)^2) : 100000")
```
that makes the constraint softer and easier to explore.
